src/components/transactions/Switch/cowprotocol.errors.ts

```diff
@@ -0,0 +1,21 @@
+const MESSAGE_MAP: { [key: string]: string } = {
+  NoLiquidity: 'No liquidity found for the given amount and asset pair.',
+  NoRoutesFound: 'No routes found with enough liquidity.',
+  SellAmountDoesNotCoverFee: 'Amount does not cover fee.',
```

Comment:
Not sure if this is clear, maybe `Sell amount is too small`, or `Amount to swap is too small` ?